docs/kf.dev/content/en/docs/developers/env/defaults/_index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Default Environment Variables"
+linkTitle: "Default Environment Variables"
+weight: 30
+description: >
+  Learn about the default environment variables available to an application in
+  Kf and how they compare to CF.
+---
+
+## Compared to CF
+
+The following table lists default environment variables available to an
+application deployed to Cloud Foundry and includes a column indicating if that
+variable is available by default in Kf.
+
+| Name                    | Available in Kf? |
+| ---                     | ---              |
+| VCAP_SERVICES           | ✔️                |
+| VCAP_APPLICATION        | 🚫               |
```

Comment:
This is available

Reply:
Huh. Here's how I pulled the envs associated with a running app (in this case, the app has no services bound):

```
kubectl get pod/spring-music-dkgcr-deployment-8588b4b58c-dkdzh -n dev -o json | jq '.spec.containers[] | select(.name=="user-container") | .env'
```

Result:

```
[
  {
    "name": "BP_JAVA_VERSION",
    "value": "8.*"
  },
  {
    "name": "KF_UPDATE_REQUESTS_bc27176d-bf87-11e9-8ddd-42010a8e00d5",
    "value": "0"
  },
  {
    "name": "PORT",
    "value": "8080"
  },
  {
    "name": "K_REVISION",
    "value": "spring-music-dkgcr"
  },
  {
    "name": "K_CONFIGURATION",
    "value": "spring-music"
  },
  {
    "name": "K_SERVICE",
    "value": "spring-music"
  }
]
```

Is `VCAP_APPLICATION` injected only in specific circumstances? Or maybe I shouldn't be looking at the pod spec?

Reply:
@evandbrown It's in a secret.
```
"envFrom": [
                    {
                        "secretRef": {
                            "name": "kf-injected-envs-manifest-app-with-service"
                        }
                    }
                ],

```
The envFrom secret mounts extra secrets, which is where VCAP_APPLICATION and VCAP_SERVICES are added to the pod.

Reply:
Ah! Got it. Table updated.